var/spack/repos/builtin/packages/nvtop/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Nvtop(CMakePackage):
+    """Nvtop stands for NVidia TOP, a (h)top like task monitor for NVIDIA GPUs.
+    It can handle multiple GPUs and print information about them
+    in a htop familiar way."""
+
+    homepage = "https://github.com/Syllo/nvtop"
+    url      = "https://github.com/Syllo/nvtop/archive/1.1.0.tar.gz"
+
+    version('1.1.0', sha256='00470cde8fc48d5a5ed7c96402607e474414d94b562b21189bdde1dbe6b1d1f3')
+
+    depends_on('cmake')
```

Comment:
This is usually not needed:
```suggestion
    depends_on('cmake')
```
unless the application needs to link some CMake internal libraries.